neurodamus/core/configuration.py

```diff
@@ -216,10 +213,8 @@ class _SimConfig:
     buffer_time = 25
     save = None
     restore = None
-    coreneuron_outputdir = None
```

Comment:
good catch! I checked and apparently I forgot to remove this line when I reworked the configuration.py. Probably due to the high-intesity back-and-forth review with Weina

Reply:
no problem